src/steps/relay.py

```diff
@@ -34,6 +40,27 @@ def setup_main_relay_nodes(self, request):
         self.node2.start(relay="true", discv5_bootstrap_node=self.enr_uri)
         self.main_nodes.extend([self.node1, self.node2])
 
+    @pytest.fixture(scope="function")
+    def register_rln_main_relay_nodes(self, request):
+        logger.debug(f"Registering RLN credentials: {inspect.currentframe().f_code.co_name}")
+        rln_creds_source = "./rln_creds.json"
```

Comment:
I would prefer to load this json for env variables. To do this we need to have it in `.env` file and with an example in `env_vars.py` (not real values just an example with the structure so the user will know what to set in .env file)
And for CI we will save it in github secrets

Reply:
Function added at https://github.com/waku-org/waku-interop-tests/pull/16/commits/6040f0875dcc02f50c2b4636a7cd6becd5c81526

Reply:
Using env variable is feasible, I am not sure how comfortable is to edit escaped string like in this commit example https://github.com/waku-org/waku-interop-tests/pull/16/commits/01d87a99334c6e25a018d4a746e81c0d64e63f1a Let me know if you still prefer this option.

Reply:
yes, I prefer to have it .env file, that way it will work the same way from .env file or from github secrets
and it should work to have json in the .env file, no need for escaping
ex:

<img width="749" alt="image" src="https://github.com/waku-org/waku-interop-tests/assets/28725451/12e267b1-d9eb-4bfa-b75e-cb863cc99729">

Reply:
if you follow those steps you can load them directly from the RLN_CREDENTIALS env var
While on CI you can achieve that by having
```
env:
  RLN_CREDENTIALS: ${{ secrets.RLN_CREDENTIALS }}
```

Reply:
Chaged at https://github.com/waku-org/waku-interop-tests/pull/16/commits/25afac4de45389ad0d6670005eae5a267be407cf